libs/lakebase/connect.go

```diff
@@ -0,0 +1,104 @@
+package lakebase
+
+import (
+	"context"
+	"fmt"
+	"os"
+	"strings"
+
+	"github.com/databricks/cli/libs/cmdctx"
+	"github.com/databricks/cli/libs/exec"
+	"github.com/databricks/databricks-sdk-go/service/database"
+	"github.com/google/uuid"
+)
+
+func Connect(ctx context.Context, databaseInstanceName string, extraArgs ...string) error {
+	fmt.Printf("Connecting to Databricks Database Instance %s ...\n", databaseInstanceName)
+
+	w := cmdctx.WorkspaceClient(ctx)
+
+	// get user:
+	user, err := w.CurrentUser.Me(ctx)
+	if err != nil {
+		fmt.Println("Error getting current user: ", err)
+		return err
```

Comment:
The additional information can be made a part of the error itself. Since the error will be printed later, we don't need to print it twice.

Same for other errors below as well.

```suggestion
		return fmt.Errorf("Error getting current user: %w", err)
```